### ISSUE TITLE
main thread panics after `exit` bash command

### ISSUE THREAD

**Issue by winterrdog** [b23r0/Cliws#2]
i tried to run `cliws`'s client side with a `/bin/sh -i` command on Kali Linux like so:
```sh
cliws -r ws://127.0.0.1:9090 /bin/sh -i 
````

But when i ran the `exit` bash command, the main thread **panicked** and displayed this code:
```sh
$ exit
thread 'main' panicked at 'Could not determine the UTC offset on this system. Possible causes are that the time crate does not implement "local_offset_at" on your system, or that you are running in a multi-threaded environment and the time crate is returning "None" from "local_offset_at" to avoid unsafe behaviour. See the time crate's documentation for more information. (https://time-rs.github.io/internal-api/time/index.html#feature-flags): IndeterminateOffset', /home/every_body/.cargo/registry/src/github.com-1ecc6299db9ec823/simple_logger-1.16.0/src/lib.rs:409:85
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

What could be the **cause** of this?

**Comment by b23r0** [comment]
I guess cause is your OS time settings incorrect , so log library can't get correct system time.

you can reset and sync your system time and time zone , if still can't solve the problem , please reply the issue.

**Comment by b23r0** [comment]
In next version , I maybe remove current log library for avoid the problem.

**Comment by winterrdog** [comment]
> I guess cause is your OS time settings incorrect , so log library can't get correct system time.
> 
> you can reset and sync your system time and time zone , if still can't solve the problem , please reply the issue.

I'll try this when I get to my machine. I'll provide you with the feedback 🤝

**Comment by winterrdog** [comment]
> > I guess cause is your OS time settings incorrect , so log library can't get correct system time.
> > 
> > you can reset and sync your system time and time zone , if still can't solve the problem , please reply the issue.
> 
> I'll try this when I get to my machine. I'll provide you with the feedback 🤝

I changed the time settings but the problem is still recurring

**Comment by b23r0** [comment]
> > > I guess cause is your OS time settings incorrect , so log library can't get correct system time.
> > > you can reset and sync your system time and time zone , if still can't solve the problem , please reply the issue.
> > 
> > 
> > I'll try this when I get to my machine. I'll provide you with the feedback 🤝
> 
> I changed the time settings but the problem is still recurring

i check the error , maybe cause is a log library bug. I fixed and updated to 0.2.2 version.
you can try the pre-release version ( https://github.com/b23r0/cliws/releases/tag/v0.2.2 ) , if the version can resolved the problem , please feedback to me.

**Comment by winterrdog** [comment]
I am going to definitely check it out. I am going to let u know

**Comment by winterrdog** [comment]
> > > > I guess cause is your OS time settings incorrect , so log library can't get correct system time.
> > > > you can reset and sync your system time and time zone , if still can't solve the problem , please reply the issue.
> > > 
> > > 
> > > I'll try this when I get to my machine. I'll provide you with the feedback handshake
> > 
> > 
> > I changed the time settings but the problem is still recurring
> 
> i check the error , maybe cause is a log library bug. I fixed and updated to 0.2.2 version. you can try the pre-release version ( https://github.com/b23r0/cliws/releases/tag/v0.2.2 ) , if the version can resolved the problem , please feedback to me.

Yes! 
It's now **fixed** and the client gets **closed nicely**. I think you can release v0.2.2 .
With that, let me close the issue.

_Nice work @b23r0 . I appreciate your time!_
